### PR TITLE
fix: update vault dto mocks

### PR DESCRIPTION
**Pull request by axejintao**

### CODE REVIEW

**Review by mrbasado — APPROVED** [review]

File: src/api/index.ts [hunk 1]
```diff
@@ -136,20 +136,6 @@ export class BadgerAPI {
     });
   }
 
-  loadCharts(
```
Comment (mrbasado) [inline src/api/index.ts:139]: not sure why this gets removed

Comment (axejintao) [reply src/api/index.ts:139]: soon to be unsupported

**Pull request merged**